modules/ipi-install-configuring-raid-for-worker-node.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-raid-for-worker-node_{context}"]
+= Configuring RAID for worker node
+
+The following procedure configures RAID (Redundant Array of Independent Disks) for the worker node during the installation process.
+
+[NOTE]
+====
+. Only servers with bmc type `irmc` are supported. Other types of servers are currently not supported.
+====
+
+.Procedure
+. Create manifests.
+. Modify the BMH (Bare Metal Host) file corresponding to the worker:
++
```

Comment:
[source,terminal] before the command per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output